asab/web/authn/oauth/middleware.py

```diff
@@ -82,7 +118,7 @@ def oauthclient_middleware_factory(app, *args, methods, identity_cache_longevity
 							"Expiration": time.time() + identity_cache_longevity
 						}
 				else:
-					raise RuntimeError("Call to OAuth server '{}' failed with status code '{}'.".format(oauth_userinfo_url, resp.status))
+					L.warn("Call to OAuth server '{}' failed with status code '{}'.".format(oauth_userinfo_url, resp.status))
```

Comment:
You really want to proceed? Is that a proper and secure reaction?

Reply:
Yes, because the `authn_required_handler` decorator then post to the user the `HTTPUnauthorized`, which is much better than the Internal Server Error, when the OAuth server refuses to provide identity.

Reply:
Can you add a comment here that explain this.